src/engine/controls/loopingcontrol.h

```diff
@@ -163,6 +163,7 @@ class LoopingControl : public EngineControl {
     ControlValueAtomic<LoopInfo> m_loopInfo;
     LoopInfo m_oldLoopInfo;
     ControlValueAtomic<mixxx::audio::FramePos> m_currentPosition;
+    ControlObject* m_pSnapEnabled;
```

Comment:
Use plain class member or std::unique_ptr. No naked `new`. I personally prefer the simplicity of a class member.